lib/spack/spack/package.py

```diff
@@ -52,26 +55,127 @@
 import spack.repository
 import spack.url
 import spack.util.web
-
-from urlparse import urlparse
-from StringIO import StringIO
 from llnl.util.filesystem import *
 from llnl.util.lang import *
 from llnl.util.link_tree import LinkTree
 from llnl.util.tty.log import log_output
+from spack import directory_layout
 from spack.stage import Stage, ResourceStage, StageComposite
 from spack.util.compression import allowed_archive
 from spack.util.environment import dump_environment
-from spack.util.executable import ProcessError, Executable, which
+from spack.util.executable import ProcessError, which
```

Comment:
I'm trying to remember whether or not Executable was important... The last time I removed something like this due to Flake8, it resulted in #1095. If a user runs Executable in their package, will they still be able to find it if you remove it from here?